src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectSecurityAwarenessScoreTest.java

```diff
@@ -19,59 +16,19 @@
   @Test(expected = IllegalArgumentException.class)
   public void noInfoAboutSecurityAdvisories() {
     PROJECT_SECURITY_AWARENESS.calculate(HAS_SECURITY_TEAM.value(false));
+    PROJECT_SECURITY_AWARENESS.calculate(USES_VERIFIED_SIGNED_COMMITS.value(false));
```

Comment:
This statement is unreachable because the first `calculate()` call throws an IllegalArgumentException. The test case should look like that:

```java
PROJECT_SECURITY_AWARENESS.calculate(
        HAS_SECURITY_TEAM.value(false), 
        USES_VERIFIED_SIGNED_COMMITS.value(false));
```

Please also update the other two test cases below.